src/scss/_functions.scss

```diff
@@ -27,44 +26,27 @@
 	@return false;
 }
 
-/// Capitalise first letter of $string
-///
-/// @param {String} $string - string to update
-///
-/// @return {String}
-@function _oFontsStringCapitalise($string) {
-	$string: $string + unquote(''); // Make sure $string has a type of String
-	$first-letter: to-upper-case(str-slice($string, 0, 1));
-	$rest-of-string: str-slice($string, 2);
-
-	@return $first-letter + $rest-of-string;
-}
-
 /// Get a font-family stack with the appropriate fallbacks
 ///
-/// @param {String} family
+/// @example scss
+/// 	font-family: oFontsGetFontFamilyWithFallbacks(FinancierDisplayWeb);
 ///
+/// @param {String} family
 /// @return {String} - font-stack
-///
-/// @example scss
-/// font-family: oFontsGetFontFamilyWithFallbacks(FinancierDisplayWeb);
 @function oFontsGetFontFamilyWithFallbacks($family) {
-	@if $family == 'Clarion' or $family == 'Benton' or $family == 'Miller' or $family == 'FinancierTextWeb' {
-		@error "#{$family} has been removed. Please use either MetricWeb or FinancierDisplayWeb.";
+	@if not map-has-key($_o-fonts-families, $family) {
+		@error 'Font "#{inspect($family)}" not found. Must be one of #{map-keys($_o-fonts-families)}.';
```

Comment:
the second sentence should maybe mention the variable name. like,

```@error 'Font "#{inspect($family)}" not found. "$family" must be one of #{map-keys($_o-fonts-families)}.';```

Reply:
👍 
```suggestion
		@error 'Font "#{inspect($family)}" not found. "$family" must be one of #{map-keys($_o-fonts-families)}.';
```